eng/Version.Details.xml

```diff
@@ -6,7 +6,7 @@
       <Sha>3dc05150cf234f76f6936dcb2853d31a0da1f60e</Sha>
       <SourceBuild RepoName="source-build-externals" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23565.1">
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.24365.2">
```

Comment:
Is it possible to align this with the latest release, or is that not new enough?  https://github.com/dotnet/dotnet/commit/8be139ddde52d33e24c7d82f813248ff9fc54b97

In general it is best to align this with the previous released version to ensure compatibility in the product source-build.

Reply:
Fixed